translate/storage/pocommon.py

```diff
@@ -205,11 +205,10 @@ class pofile(poheader.poheader, base.TranslationStore):
     # We don't want windows line endings on Windows:
     _binary = True
 
-    def __init__(self, inputfile=None, encoding=None, duplicatestyle=None):
-        super(pofile, self).__init__(unitclass=self.UnitClass)
+    def __init__(self, inputfile=None, duplicatestyle=None, **kwargs):
```

Comment:
I am worried about these changes in the signatures. The order is altered and the positional `encoding` is now a keyword argument. Please note that this might be used in third party software.